*** run_prettify.js.orig	Tue Mar  5 05:12:24 2013
--- run_prettify.js	Sat Jan 24 16:36:43 2015
***************
*** 175,180 ****
--- 175,188 ----
    var LOADER_BASE_URL =
       'https://google-code-prettify.googlecode.com/svn/loader';
  
+    // DokuWiki Plugin Code Prettifier
+    // by Satoshi Sahara <sahara.satoshi@gmail.com>
+    // 2015-01-24
+    // Added 3 lines to accommodate the Auto-loader for DokuWiki
+    if (typeof JSINFO.plugin_codeprettify.loader_base !== undefined) {
+        LOADER_BASE_URL = JSINFO.plugin_codeprettify.loader_base;
+    }
+ 
    for (var i = 0, n = langs.length; i < n; ++i) (function (lang) {
      var script = doc.createElement("script");
  
